### DISCUSSION TITLE
Welcome to AdashimaaTube Discussions!

### DISCUSSION THREAD

**Discussion by sapondanaisriwan** [sapondanaisriwan/AdashimaaTube#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Myphie** [comment]
Hi! I was wondering if it's possible to make a WATCHED overlay for youtube videos. Have been looking for one that works.

**Reply by sapondanaisriwan** [reply]
Do you mean miniplayer?
![image](https://user-images.githubusercontent.com/64634605/208303541-500c7387-8e3f-4ed0-b970-ed41a50c3c5b.png)

**Reply by Myphie** [reply]
ah, no. Like an indicator over the thumbnail to indicate that you’ve already watched the video. 
![60B655E1-DFB5-4F5B-85F5-BEF90D327A9B](https://user-images.githubusercontent.com/46834071/208304120-5663956a-cf38-442a-b2f5-7ba25f5e69ab.jpeg) like this

**Reply by sapondanaisriwan** [reply]
I'll try it out with just CSS. I am not sure if it is possible or not.

**Reply by sapondanaisriwan** [reply]
I did it
![image](https://user-images.githubusercontent.com/64634605/208434797-fda4cd0b-b197-4c66-a2d0-d6bc2f421f17.png)

**Reply by Myphie** [reply]
Awesome! Thank you so much! (:

**Comment by SamGitHub1807** [comment]
Hello! I'm Sam. I absolutely love this project! Not a big fan of the new Youtube layout, so that's why I'm using this, and also because you have your own additions! Furthermore, here's another Idea I'd like to see!

(I'm not sure if this is already being worked on!)
(This is a recent change that happened on: I believe around late 2022. And I'm also not sure if this change has rolled out for you yet but it has for me.)

Return the Red "HD/4K/8K, etc" icons inside the YouTube quality picker.

Before
<img width="263" alt="Screenshot 2023-01-14 at 20 32 17" src="https://user-images.githubusercontent.com/122677421/212471713-4fe7c76e-5018-4da2-82a7-7292ffb2393f.png">

After
<img width="209" alt="Screenshot 2023-01-14 at 20 24 09" src="https://user-images.githubusercontent.com/122677421/212471709-09d9ecb8-96e3-406d-8a61-8e3c21acd891.png">

Oh, forgot about when you click settings for the first time but not open the quality picker.

Before
<img width="198" alt="Screenshot 2023-01-14 at 20 35 05" src="https://user-images.githubusercontent.com/122677421/212471773-83ff906a-a331-4799-bdfc-5c7ac2ea4629.png">

After
<img width="232" alt="Screenshot 2023-01-14 at 20 36 56" src="https://user-images.githubusercontent.com/122677421/212471839-3086bebe-8753-445b-ba1f-52de8445e77e.png">

They added parentheses.

For reference, here's what it looked like before if it's on auto with an HD quality.
<img width="1110" alt="Screenshot 2023-01-14 at 21 27 00" src="https://user-images.githubusercontent.com/122677421/212474153-e0e3857a-208a-4f2c-97fb-cf7f2b38fe9f.png">

VS what it looks like now.
<img width="227" alt="Screenshot 2023-01-14 at 21 39 02" src="https://user-images.githubusercontent.com/122677421/212474717-783e2249-064d-4ef9-ac7c-38aa9828821f.png">


By the way, if you picked the quality manually, then the parentheses won't show up, only when it's on auto.
<img width="200" alt="Screenshot 2023-01-14 at 21 37 26" src="https://user-images.githubusercontent.com/122677421/212474673-30845457-3a17-4c9a-a24f-2e921b9673eb.png">


This is an extremely minor request, so you guys don't have to do it! I don't know much about coding so I really don't know how difficult this is to do, but it'd be awesome if you could incorporate something like this!

**Reply by sapondanaisriwan** [reply]
Thanks for the feedback. What I can do right now is change the color of "HD/4K/8k," but I'm not sure if the parentheses can be removed using only CSS.
![image](https://user-images.githubusercontent.com/64634605/212480531-fa5bb37a-be8f-4ac8-8eaf-653fd2a4bf62.png)

**Reply by SamGitHub1807** [reply]
Wow! It’s perfect. Even if the parentheses cannot be removed, this is what I was hoping for nevertheless! (because YouTube's design choices are weird, it being gray made it boring lol) Thank you!

**Reply by sapondanaisriwan** [reply]
No problem 😄

**Reply by SamGitHub1807** [reply]
You’re welcome!

**Reply by sapondanaisriwan** [reply]
Remove parentheses: [e5ad2b8](https://github.com/sapondanaisriwan/AdashimaaTube/commit/e5ad2b8a6bd583097b97f039695c17ec1adfc174)

**Comment by SamGitHub1807** [comment]
nevermind lol this is too much! i don't want to stress you

**Reply by sapondanaisriwan** [reply]
> nevermind lol this is too much! i don't want to stress you

I don't find it to be too much for me. I forgot to tell you, you have to creating an [issue](https://github.com/sapondanaisriwan/AdashimaaTube/issues/new)  to request the feature or leave a suggestion.

**Reply by SamGitHub1807** [reply]
I see. I'll bear this in mind if I do another bug report or feature request/s. I pretty much just edited my bug report for the user interface, etc.. and changed it into "it's too much and I don't want to stress you."

**Comment by Jurci11** [comment]
Hello, Its possible remove empty spaces on sides of channel page and scale size of content  base on number in row setting ?
Thank you

![image](https://user-images.githubusercontent.com/60761228/217498273-d75c2247-fbea-42e6-a68b-991e190c4921.png)

**Reply by sapondanaisriwan** [reply]
> Hello, Its possible remove empty spaces on sides of channel page and scale size of content base on number in row setting ? Thank you
> 
> ![image](https://user-images.githubusercontent.com/60761228/217498273-d75c2247-fbea-42e6-a68b-991e190c4921.png)

I think it's possible but normal Youtube is the best

**Comment by SamGitHub1807** [comment]
Hey! Sam again, this time I made a discussion for something since I think it would be difficult to achieve, so I won't even make an issue hahaha. Anyways! With the new v1.9.5 update that fixed the overlapping elements (**yes, I waited for this update haha since if you corrected the spacing, it would cause issues, specifically for those with smaller screen resolutions**). Anyways, continuing, do you think it's possible to change the spacing of the watch page buttons to reflect YouTube's official spacing? Here are some photos:

YouTube's official spacing (October 1, 2021):

<img width="324" alt="Screenshot 2023-02-27 at 07 02 15" src="https://user-images.githubusercontent.com/122677421/221442868-61cc7862-0e97-4bac-adbc-132290a2401c.png">

The style:
<img width="317" alt="Screenshot 2023-02-27 at 06 52 34" src="https://user-images.githubusercontent.com/122677421/221442874-7b3d648f-69f0-4392-bf4b-7fcc39337a51.png">

Here's another official YouTube layout screenshot, but this time it's when they added the clip, download, etc, buttons lol. 
<img width="869" alt="image" src="https://user-images.githubusercontent.com/122677421/221442927-66aa0cbb-e2aa-4706-b50d-2031296d2cc0.png">

If you do this you'd also have to change the length of the ratio-bar because of the spacing as seen in the previous 2 screenshots That's why I thought it'd be hard to achieve. A similar issue by @ PANMARSAG14 was made before this, I just thought I'd give more information and also add that the like and dislike button also had incorrect spacing. 

Also since I'm doing this discussion, I might as well put really minor changes and fixes here instead of making a new issue. 

1: This drop-down menu when you click the three dots and the quality picker had a little bit of roundness, until YouTube's new layout that made it more round lol.
<img width="174" alt="image" src="https://user-images.githubusercontent.com/122677421/221491969-3658ba65-c558-400c-9998-455b6948729a.png">
<img width="648" alt="image" src="https://user-images.githubusercontent.com/122677421/221443409-5b0f7e35-f305-4eb5-84b0-c357de0f71bb.png">

2: The hashtags above the title fix changed the spacing a bit, so now, even videos without hashtags have an empty space where the hashtags would be.

Videos without hashtags:

YouTube's official spacing:
<img width="230" alt="image" src="https://user-images.githubusercontent.com/122677421/221491466-1729400b-f0b4-43a6-b918-5005434ff727.png">

The style:
<img width="296" alt="image" src="https://user-images.githubusercontent.com/122677421/221490630-78ab2b32-8bc7-482e-be09-1041915f7ad0.png">

Videos with hashtags:

YouTube's official spacing:
<img width="206" alt="image" src="https://user-images.githubusercontent.com/122677421/221490449-74a4b70a-be5c-4f96-afec-29b7fbacbf66.png">

The style:
<img width="283" alt="image" src="https://user-images.githubusercontent.com/122677421/221490393-67f92826-f04a-46ae-b5e9-d63085ee7c5b.png">

More suggestions that I think are hard/impossible to achieve:

If you observe the ratio bar, it had no roundness. (by the way this was YouTube's official ratio bar not Return YouTube Dislike!)
Another thing is that, the ratio bar was closer to the border line, I think it's actually on top of the border line.

That's all and thank you!

**Reply by sapondanaisriwan** [reply]
Please don't be hesitant to report any bugs you find to me. like the overlapping elements that I just found yesterday.

**Reply by sapondanaisriwan** [reply]
Moved to [issues](https://github.com/sapondanaisriwan/AdashimaaTube/issues/27)

**Comment by IdleCommentator** [comment]
My personal feedback regarding 1.9.7 - this is the first time I had to roll back to a previous version of the style. Personally, I find your work on this style pretty impressive as well as the way this userstyle became better over time with updates - however, for me unfortunately the simplifications and feature removals made in latest version took out some important functionality that made using it discernibly worse.
I'm not sure whether I'm in a small minority here - but not having separate settings for Youtube homepage and channel pages as well as "Sort by Oldest" being gone was a rather big deal for me, hence having to go back to 1.9.6.
Though, I guess, as a tradeoff I'll have to deal with somewhat misaligned look of channel pages in 1.9.6 as a result of latest Youtube UI update for now.

**Reply by sapondanaisriwan** [reply]
I'll bring the features that I removed back in the next coming update.

**Reply by MossyMosMos** [reply]
"Sort by Oldest" function at "Channel Page" option has returned in version 1.9.8 or 1.9.9 (latest version)

**Reply by sapondanaisriwan** [reply]
Both I suggest use the latest version

**Comment by Lunarhawk** [comment]
After youtubes recent UI update that lowered the vids per row to 3 on the video/live pages I got this to raise it back up to an amount that didn't look completely awful however I've noticed it "breaks" some things on the /watch page for live streams and you can't actually disable the stylus completely for just /watch to fix them 
Specifically it breaks the counters for watching now and started streaming as seen here
![image](https://user-images.githubusercontent.com/4963241/225183366-f4078fcc-10d6-4c24-99b5-1f6871f27464.png)

as they're supposed to auto refresh/update appropriately but they don't with the stylus active and are locked to whatever numbers they are when you open the stream. Is it possible to either fix these counters or, if easier, make disabling "watch" page in the settings actually disable it entirely for that page so youre seeing the current kind of crummy watch page layout but at least those counters still work?

**Reply by sapondanaisriwan** [reply]
> After youtubes recent UI update that lowered the vids per row to 3 on the video/live pages I got this to raise it back up to an amount that didn't look completely awful however I've noticed it "breaks" some things on the /watch page for live streams and you can't actually disable the stylus completely for just /watch to fix them Specifically it breaks the counters for watching now and started streaming as seen here ![image](https://user-images.githubusercontent.com/4963241/225183366-f4078fcc-10d6-4c24-99b5-1f6871f27464.png)
> 
> as they're supposed to auto refresh/update appropriately but they don't with the stylus active and are locked to whatever numbers they are when you open the stream. Is it possible to either fix these counters or, if easier, make disabling "watch" page in the settings actually disable it entirely for that page so youre seeing the current kind of crummy watch page layout but at least those counters still work?

Unfortunately, I can't fix that, but the most recent update, version 2.0.8, allows you to switch between the old and the new layout.
![o07QQslcZt](https://user-images.githubusercontent.com/64634605/225346617-9894ec0d-8572-4b73-b349-ddf5bfe62243.png)

**Reply by Lunarhawk** [reply]
Thank you for the quit update! I just got this a few days ago at 2.0.1 so I didn't think I'd have to check for an update so quickly this works perfectly for what I wanted
